projects/ion/src/lib/tab/tab.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Input } from '@angular/core';
+
+export type Size = 'sm' | 'md' | 'lg';
```

Comment:
```suggestion
export type TabSize = 'sm' | 'md' | 'lg';
```

change this not to confuse with chip size.